docs/terraform.md

```diff
@@ -21,6 +21,7 @@
 | delimiter | Delimiter to be used between `name`, `namespace`, `stage`, etc. | string | `-` | no |
 | enabled | Select Enabled if you want CloudFront to begin processing requests as soon as the distribution is created, or select Disabled if you do not want CloudFront to begin processing requests after the distribution is created. | string | `true` | no |
 | forward_cookies | Time in seconds that browser can cache the response for S3 bucket | string | `none` | no |
+| forward_header_values | A list of whitelisted header values to forward to the origin | list | `<list>` | no |
```

Comment:
nit: `forward_header_values` is not consistently named relative to the other parameters in this group. Should simply be `forward_headers`